safe/gui/tools/wizard/step_fc90_analysis.py

```diff
@@ -259,3 +261,54 @@ def progress_callback(self, current_value, maximum_value, message=None):
         self.progress_bar.setMaximum(maximum_value)
         self.progress_bar.setValue(current_value)
         QtGui.QApplication.processEvents()
+
+    def print_map(self):
+        """Open impact report dialog used to tune report when printing."""
+        # Check if selected layer is valid
+        impact_layer = self.parent.iface.activeLayer()
+        if impact_layer is None:
+            # noinspection PyCallByClass,PyTypeChecker
+            QtGui.QMessageBox.warning(
+                self,
+                self.tr('InaSAFE'),
```

Comment:
I guess we are calling `tr('InaSAFE')` in many places, so it will not affect translations